src/components/PhotoShootView/PhotoShootView.tsx

```diff
@@ -45,27 +56,34 @@ const PhotoShootView: React.FC<PhotoShootData> = ({
     setLocation('')
   }
 
-// Will change the photo with the carousel (need more info from the api to get it to work)
-	// const displayCarousel = () => {
-	// 	const currentPhoto = photoList.find((photo: any) => photo.photoNumber === carousel + 1)
-	// 	const participants = currentPhoto.guests.reduce((guest:any, acc:any) => {
-	// 		const match = guests.find((person: any) => person.id === guest)
-	// 		acc.push(match.name)
-	// 		return acc
-	// 	}, [])
-	// 	return (
-	// 		<StyledCard contents="photoShoot">
-	// 			<article className="photoShootCard">
-	// 				<Photo
-	// 					id={currentPhoto.id}
-	// 					photoNumber={currentPhoto.photoNumber}
-	// 					guests={participants}
-	// 					description={currentPhoto.description}
-	// 				/>
-	// 			</article>
-	// 		</StyledCard>
-	// 	)
-	// }
+	const displayCarousel = () => {
+		const currentPhoto = photoList.find((photo: any) => photo.number === carousel + 1)
+		if(currentPhoto) {
+			const participants:string[] = []
+			const filtered = currentPhoto.guest.forEach((num:any) => {
+				const match = guests.find((person: any) => person.id === num)
+				if (match) {
+					participants.push(match.name)
+				}
+			})
+			if(participants.length > 0) {
+				return (
+					<StyledCard contents="photoShoot">
+					<article className="photoShootCard">
+					<Photo
+					id={currentPhoto.id}
+					photoNumber={currentPhoto.number}
+					guests={participants}
+					description={currentPhoto.description}
+					/>
+					</article>
+					</StyledCard>
+				)
+			}
+		} else {
+			return "Great job! You're done!"
+		}
+	}
```

Comment:
This carousel feature is awesome!

Reply:
The buttons aren't quite responding the way I wanted them too, so we'll add that to our list of 'things to grow on' lol